qutip/qip/device/modelprocessor.py

```diff
@@ -0,0 +1,357 @@
+# This file is part of QuTiP: Quantum Toolbox in Python.
```

Comment:
Yes that was indeed our discussion on Monday. But I then realized that since we call the whole sub-module `device`, calling this class `DeviceProcessor ` sounds like it is the base class in this module, which is not true. The base class is called `Processor`.